src/EditorFeatures/VisualBasic/LineCommit/CommitBufferManager.vb

```diff
@@ -97,63 +96,119 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
         ''' the formatting span to an entire block
         ''' </summary>
         Public Sub CommitDirty(isExplicitFormat As Boolean, cancellationToken As CancellationToken)
+            CommitDirtyAsync(isExplicitFormat, blocking:=True, cancellationToken).Wait(cancellationToken)
```

Comment:
this is a bit ugly, but it matches what we do elsewhere when we have a core routine we want to share among a totally async consumer and a totally sync consumer.  the consumer passes in the style they want, and the impl adjusts accordingly (for example, calling GetSyntaxTreeSynchronously so that it can complete asap on the UI thread).